### ISSUE TITLE
regex cannot match google fonts

### ISSUE THREAD

**Issue by alfadb** [AaronJan/postcss-font-grabber#1]
Google fonts like:
```
@font-face {
  font-family: 'Source Sans Pro';
  font-style: normal;
  font-weight: 700;
  src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'), url(https://fonts.gstatic.com/s/sourcesanspro/v9/toadOcfmlt9b38dHJxOBGLsbIrGiHa6JIepkyt5c0A0.ttf) format('truetype');
}
```
Cannot match with your regex pattern(src/lib/regexes.js).

**Comment by AaronJan** [comment]
Got this 👍 , will fix this soon.
Thanks for your feedback!

**Comment by AaronJan** [comment]
It's fixed in `1.0.6`, have a try. 😄
